letsdrill.rb

```diff
@@ -1,12 +1,19 @@
 def get_letter_grade(integer)
  
   #Put your code here!
-
+  case integer
+    when 90..100 then "A"
```

Comment:
Look up documentation on whatever text editor you're using to see if you can adjust how many spaces the Tab key uses.